extensions/amp-video/0.1/test/test-flexible-bitrate.js

```diff
@@ -212,6 +212,16 @@ describes.fakeWin('amp-video flexible-bitrate', {}, (env) => {
         })
       ).to.jsonEqual([null, null, null, null, null, null, 'CACHE', null, null]);
     });
+
+    it('should sort sources only when it\'s not already sorted', () => {
+      const m = getManager('4g');
+      const v0 = getVideo([4000, 1000, 3000, 2000]);
+      m.manage(v0);
+      
+      // Sorting once should work, but second time it should be a noop.
+      expect(m.sortSources_(v0)).to.be.true;
+      expect(m.sortSources_(v0)).to.be.false;
+    });
```

Comment:
Coverage from this test is poor, it only tests your `sortSources_()` return value, but nothing of what you do with it. Please write a test that ensures we don't get into a `load()` loop, eg: "trying to downgrade when there is no lower bitrate doesn't call `load()`"